src/main/webapp/i18n/en/global.json

```diff
@@ -186,7 +186,10 @@
             "cleanup": "Cleanup",
             "archive": "Archive",
             "submit": "Submit",
-            "submitDeadlineMissed": "Submit (deadline missed)",
+            "submitTooltip": "You can submit multiple times. Your last submission will count towards your score.",
+            "submitDeadlineMissedTooltip": "The deadline has expired. Your submissions will not get graded!",
+            "deadlineMissedTooltip": "The deadline has expired. You can no longer submit any solutions!",
+            "submitDeadlineMissed": "Submit (deadline has expired)",
```

Comment:
Please, revert it `submitDeadlineMissed` translation

Reply:
Reverted